dev_settings.py

```diff
@@ -90,7 +90,7 @@
 PILLOWTOP_MACHINE_ID = 'testhq'  # for tests
 
 #  make celery synchronous
-CELERY_ALWAYS_EAGER = True
+CELERY_TASK_ALWAYS_EAGER = True
```

Comment:
Anyone who has this overridden in their localsettings.py that will probably want to change their override.